packages/developer-portal/src/components/apps/pipeline/dns/dns-settings-page.tsx

```diff
@@ -57,6 +29,13 @@ export const DnsSettingsPage: FC<{
             notify your relevent Dev Ops team about the records mentioned below.
           </BodyText>
         </InputWrapFull>
+        <InputWrapFull>
```

Comment:
Need to rethink this sentence:

`   Your custom DNS setup was completed on IaaS. For your custom domain to be working, you will now you need to
            notify your relevent Dev Ops team about the records mentioned below.`

The first part suggests it's already been set up. Maybe something like

`Setup a custom domain name for your deployment from this page. To be able to complete the setup, the relevant DevOps team will need to accept the records below. The pipeline will create a PR automatically for their review`